src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (inverse[value] !== undefined) {
```

Comment:
Replace with `inverse[value]`, it will be the same as `inverse[value] !== undefined`